### DISCUSSION TITLE
Burp Cert in Nuclei

### DISCUSSION THREAD

**Discussion by shelu16** [projectdiscovery/nuclei#3866] [Q&A]
Hi, Can I use the Burp certificate in nuclei  ?
if yes then how

**Comment by Mzack9999** [comment]
@shelu16, could you elaborate on which certificate you are referring to? The one under settings->tools->proxy? 
Generally, burp suite performs outgoing tls connection using java tls, whereas the burp certificate is used only for the internal proxy server unless specific client certificates are specified for targets.

**Comment by shelu16** [comment]
but some time  when i try to access domain , i am not able to access but in burp its working

**Comment by Mzack9999** [comment]
It might not be related to certificate. We need reproduction steps in order to investigate the issue. Please fill all the fields of the [issue template](https://github.com/projectdiscovery/nuclei/issues/new?assignees=&labels=Type%3A+Bug&projects=&template=issue-report.md)

**Comment by shelu16** [comment]
ok

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
